docker-compose.yml

```diff
@@ -1,14 +1,19 @@
 db:
   image: postgres:9.4
+  ports:
+     - "5432:5432"
+  environment:
+     - POSTGRES_HOST_AUTH_METHOD=trust
+
 web:
   build: .
   command: python systers_portal/manage.py runserver 0.0.0.0:8000
   volumes:
-    - .:/usr/src/portal
+     - .:/usr/src/portal
```

Comment:
Any reasons for changing this?

Reply:
The extra space has actually no specific reason . I have just re-written the dockere.yml so to maintain proper indentation I guess have added extra space.

Reply:
Generally. indentation is done in multiples of 2, could you change this back?

Reply:
Ok, sure I am actually not aware of indentation rule.